lib/stat_tracker.rb

```diff
@@ -181,18 +191,48 @@ def coach_stats
     end
     coach_percentages
   end
+  
+  def find_season(season_id)
+    @seasons.find do |season|
+      season.season == season_id
+    end
+  end
 
-  def winningest_coach
-    best_coach_stats = coach_stats.max_by {|coach, percentage| percentage}
+  def winningest_coach(season)
+    best_coach_stats = coach_stats(season).max_by {|coach, percentage| percentage}
     best_coach_stats[0]
   end
 
-  def worst_coach
-    worst_coach_stats = coach_stats.min_by {|coach, percentage| percentage}
+  def worst_coach(season)
+    worst_coach_stats = coach_stats(season).min_by {|coach, percentage| percentage}
     worst_coach_stats[0]
   end
 
-  def most_accurate_team
-    
+  def most_accurate_team(season)
```

Comment:
Good job on these methods!
I was thinking a parameter needed to be passed in like season!